tests/rados/test_osd_scrub_load_threshold_parameter.py

```diff
@@ -0,0 +1,548 @@
+        ), "Could not set pg_autoscale mode to OFF"
+
+        if not rados_object.create_pool(**replicated_config):
+            log.error("Failed to create the replicated Pool")
+            return 1
+
+        rados_object.bench_write(pool_name=pool_name, byte_size="5K", max_objs=10000)
+        msg_data_push = f"The data pushed into the {pool_name} pool"
+        log.info(msg_data_push)
+
+        # Get the pool pg_id
+        pg_id = rados_object.get_pgid(pool_name=pool_name)
+        pg_id = pg_id[0]
+        msg_pool_id = f"The {pool_name} pg id is - {pg_id}"
+        log.info(msg_pool_id)
+
+        # Get the acting PG set
+        acting_pg_set = rados_object.get_pg_acting_set(pool_name=pool_name)
+        msg_acting_set = f"The {pool_name} pool acting set is -{acting_pg_set}"
+        log.info(msg_acting_set)
+
+        # Check that the scrubbing is progress or not
+        scrub_object.wait_for_pg_scrub_state(pg_id, wait_time=wait_time)
+
+        default_threshold_value = mon_obj.get_config(
+            section="osd", param="osd_scrub_load_threshold"
+        )
+
+        if float(default_threshold_value) != 10.0:
+            log.error(
+                "The default value of the osd_scrub_load_threshold is not equal to 10.0"
+            )
+            return 1
+        log.info("The default value of the osd_scrub_load_threshold is 10.0")
+
+        log.info(
+            "===== Scenario 1: Verification of the osd_scrub_load_threshold feature with the value 0"
+        )
+        log.info(
+            "Scenario 1: Setting the osd_scrub_load_threshold parameter value to 0"
+        )
+        # Perform the tests by modifying the
+        mon_obj.set_config(section="osd", name="osd_scrub_load_threshold", value=0)
+
+        # Truncate the osd logs
+        rados_object.remove_log_file_content(osd_nodes, daemon_type="osd")
+        log.info("Scenario1: The content of osd log are removed")
+        # set the debug log to 20
+        mon_obj.set_config(section="osd", name="debug_osd", value="20/20")
+        log.info("Scenario1: The osd and mgr logs are set to 20")
+        init_time, _ = installer.exec_command(
+            cmd="sudo date '+%Y-%m-%dT%H:%M:%S.%3N+0000'"
+        )
+        init_time = init_time.strip()
+
+        set_scheduled_scrub_parameters(scrub_object, acting_pg_set)
+
+        try:
+            chk_deep_scrub = rados_object.start_check_deep_scrub_complete(
+                pg_id=pg_id, user_initiated=False, wait_time=wait_time
+            )
+        except Exception:
+            log.info(
+                "Scrub operation not started after setting the osd_scrub_load_threshold value to 0"
+            )
+            chk_deep_scrub = False
+
+        if chk_deep_scrub:
+            log.error(
+                "Scrub operation started after setting the osd_scrub_load_threshold value to 0"
+            )
+            return 1
+        # Check
+        mon_obj.remove_config(section="osd", name="debug_osd")
```

Comment:
remove the configs in places where it was set

Reply:
Introduced new method to set or unset the osd log.